frontend/config/webpack.config.dev.js

```diff
@@ -16,10 +16,8 @@ module.exports = merge(webpackBaseConfig, {
     host: '0.0.0.0',
     historyApiFallback: true
   },
-  plugins: [
-    new webpack.NamedModulesPlugin()
-  ],
+  plugins: [new webpack.NamedModulesPlugin()],
   performance: {
-    hints: 'warning'
+    hints: false
```

Comment:
not related, but this is annoying as hell, so I disabled it

Reply:
![fine](http://s2.quickmeme.com/img/e0/e0f3ecd64e56f6287b3bbaa14fcac96057b7e2494190d046ec32f981f3de5762.jpg)